utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -15,6 +15,10 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
+# A mapping from license strings collected from the declared licenses of Open Source packages to SPDX expressions. This
+# mapping only contains license strings which can *not* be parsed by [SpdxExpression.parse], for example because the
+# license names contain white spaces. See [SpdxSimpleLicenseMapping] for a mapping of varied license names.
```

Comment:
Should it be various instead of varied?

Reply:
No, "varied" is correct (and also used in the original text that I copied this from) as it's supposed to mean "variants of license names" and not "several license names".